packages/vue/examples/data-search/src/main.js

```diff
@@ -0,0 +1,10 @@
+import Vue from "vue";
```

Comment:
Linting is messed up. You are using 2 spaces for indentation. Throughout this project, we use 4-width tabs.